addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/cgate/CGateConnectException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *  CGateInterface - A library to allow interaction with Clipsal C-Gate.
```

Comment:
Hi Guys, Ive been following the progress... Thank Dave for relicensing your great library... I have some bug fixes (mainly fixes the return event not working) for the binding and Kai's suggestions, will post once tested this weekend... unless Scott gets to it first..

Simon